BestFloww/BestFlow
# Creating intentdiagram

## Pull Request Summary
Added a semi-styled IntentDiagram component meant to be used to list the various questions intent by intent. Also made small edit to MainPage.jsx testing.

### Change types


### Testing


### How to manually test this PR (QA instructions)
Go to main page and ensure the element below renders. Hover over the branches and make sure the outline changes. Use tab to select the branches and make sure the outline changes (a11y check)
![image](https://user-images.githubusercontent.com/76976842/196063646-6cc8d329-6e89-41a2-b489-5b41ab9649e2.png)

Try changing around the demo element in MainPage.jsx and make sure it does not break (under reasonable constraints)

Also LMK if you have design suggestions :)
